src/components/ble/DfuService.cpp

```diff
@@ -165,10 +165,10 @@ int DfuService::WritePacketHandler(uint16_t connectionHandle, os_mbuf* om) {
 
       if ((nbPacketReceived % nbPacketsToNotify) == 0 && bytesReceived != applicationSize) {
         uint8_t data[5] {static_cast<uint8_t>(Opcodes::PacketReceiptNotification),
-                         (uint8_t)(bytesReceived & 0x000000FFu),
-                         (uint8_t)(bytesReceived >> 8u),
-                         (uint8_t)(bytesReceived >> 16u),
-                         (uint8_t)(bytesReceived >> 24u)};
+                         (uint8_t) (bytesReceived & 0x000000FFu),
+                         (uint8_t) (bytesReceived >> 8u),
+                         (uint8_t) (bytesReceived >> 16u),
+                         (uint8_t) (bytesReceived >> 24u)};
```

Comment:
There may be a difference between `clang-format` versions that's causing the error here. We should be using static_cast anyway, so let's change them now.

Reply:
Yep, I wanted to highlight that C-style casts should be replaced as well when formatting.